### DISCUSSION TITLE
Audi MHI2Q Navigation stuck in Initializing 11% / and Parking Distance assist wont work

### DISCUSSION THREAD

**Discussion by elflako17** [Mr-MIBonk/M.I.B._More-Incredible-Bash#448] [Q&A]
Is there a way to fix this? I installed MIB, patched the software, enabled VIM, and enabled the navigation to work still if carplay navigation is on. after that, i updated the maps to 2024 which worked for 1 day and now I am getting this error

![camphoto_824023566](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/43461165/878fc12c-906a-453b-867d-ad0c4449db80)
![IMG_2199](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/43461165/d4614ff8-2091-4a76-9fc6-a387289c0f46)
![IMG_2205](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/43461165/2d81caa6-bcd4-4385-9281-03e477641ffa)
![IMG_2210](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/43461165/bf50a73c-1565-46bf-8821-3f7690a9563c)

**Comment by AldoBuzi** [comment]
Did you find any solution for the problem? Mine does the same thing :(

**Reply by elflako17** [reply]
i had some one fix it for me remotely.

**Reply by bwilliams75** [reply]
Who was able to fix this for you and how did they do it? I am currently having this issue with my sq5.

**Comment by fuadamin** [comment]
Hi, this problem happens when your car info is not properly set, that is why map is not loading and there is no PDC view and there is no car. I have same problem and fix by using MIB toolbox. 
To do that, use model_db from mods folder and find the code for audi q2 (01 72 06)
01 - manufacturing company, 01 is audi.
72 - here 7 is car generation and 2 is car class.
![20240716_173150](https://github.com/user-attachments/assets/038547f5-1c3d-42ec-abe2-3edd2c0c35c4)

06 - here 0 is car derivate and 6 is car derivate supplement .
![20240716_173159](https://github.com/user-attachments/assets/16a15872-61f3-4bd8-8ed0-027b805f8670)

** do not follow the picture just follow the text**

**Reply by Landy989** [reply]
I am also experiencing the same issue with my 2019 TT. Could you help me with this please or point me in the right direction?

**Reply by fuadamin** [reply]
You have to set vehicle coding 
013304|Audi TT (8S0)
013305|Audi TT (8S0)
013354|Audi TTS (8S0)

**Reply by Landy989** [reply]
Thank you! What is the difference between 04 and 05?

**Reply by fuadamin** [reply]
Just try look in the manual 
![20240716_173159](https://github.com/user-attachments/assets/56b3393c-36ba-46ea-93ed-0e4e918bdeb0)
Or
 Try find the pr code of your  car 
PR code looks like

![image](https://github.com/user-attachments/assets/f6cdacac-c344-4b74-ab5d-894fbd650dad)

**Reply by n2ckjansens** [reply]
> Just try look in the manual ![20240716_173159](https://private-user-images.githubusercontent.com/26693511/354811157-56b3393c-36ba-46ea-93ed-0e4e918bdeb0.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwNTU5OTYsIm5iZiI6MTcyNDA1NTY5NiwicGF0aCI6Ii8yNjY5MzUxMS8zNTQ4MTExNTctNTZiMzM5M2MtMzZiYS00NmVhLTkzZWQtMGU0ZTkxOGJkZWIwLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDA4MjEzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBkZWU4MzVjNTkyNzBjMDM5NGZmMzkwNmU5YTQyNDA2YzliMTIwZmEzODcxMDg4NmIwMmVlMTUwZWY0YmYzMWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.DZ2gU37LHqhUJedohCBx6vmrHiDxy5w0SXRgApJcd_M) Or Try find the pr code of your car PR code looks like
> 
> ![image](https://private-user-images.githubusercontent.com/26693511/354811391-f6cdacac-c344-4b74-ab5d-894fbd650dad.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwNTU5OTYsIm5iZiI6MTcyNDA1NTY5NiwicGF0aCI6Ii8yNjY5MzUxMS8zNTQ4MTEzOTEtZjZjZGFjYWMtYzM0NC00Yjc0LWFiNWQtODk0ZmJkNjUwZGFkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDA4MjEzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVjYjhiNDJiMWJkMjcwYmY5OTY0MzA0ZDc3MjVjYzFlYmU2NmY1ZWY0ODhiY2Q1YzhlYmI4NWIxYjQ1YWY5MjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Q-gHq8cSx0jSVjQ4G-Y3pGYA6BQa5iEgsaU9CQTL0Dk)

I am also experiencing the same issue with my 2018 A3 e-tron. Can't find code about you told in this post. Could you help me with this please or point me in the right direction?

**Reply by Tomt97** [reply]
> You have to set vehicle coding 013304|Audi TT (8S0) 013305|Audi TT (8S0) 013354|Audi TTS (8S0)

Anyone know for 2018 q5

**Reply by PepeBarcelona** [reply]
@n1ckjansens , did you find any solution for issue? I also have an Audi A3 e-tron (2017 facelift) with the very same problem which came up after a damaged maps update. I have already checked bytes 0, 1, 2, and 3 in my the long coding of my 5F module and compared them with "model_db" file, but everything is ok.
Currently, I am considering is to connect my computer via ethernet (using a dub e100 adapter), log in via ssh and run the following commands (which were suggested by FPSUsername in reddit:

**rm -Rf /net/mmx/mnt/gecache/***
This command recursively deletes everything inside /net/mmx/mnt/gecache/ without confirmation.

**rm -f /net/mmx/mnt/adb/truffles***
This command deletes all files starting with "truffles" in the specified directory without confirmation.

**on -f mmx sync**
This command is likely used to run the synchronization status of the mmx system or service.

**Then reboot.**

Any suggestion?

**Reply by lalley2ka** [reply]
@PepeBarcelona , this worked spectacularly for me!

I had a problem a similar to @n1ckjansens on a stock radio on a 2017 Cayenne.  Navigation initially was just slow to start, and then got worse and worse until it wouldn't start at all.  Evidently I had corruption in my gecache or truffles database.  I installed the sshd server from the MIB2 toolbox, and I was able to put in the commands by connecting my car to my home wifi even without having the DUB E100 adapter.  Problem solved!  

Thanks so much for mentioning this, and thanks to FPSUsername, whoever they are!

**Reply by RawBee4572** [reply]
@PepeBarcelona the solution proposed worked perfectly for me too 🙏 although I've cleared only `gecache`, `truffles` were already empty for me

**Reply by Woody8219** [reply]
Hi, which was the solution for this? I have the 11 percent prob. Thanks in advance!

**Reply by rynstphn** [reply]
model_db doesn’t include A4 allroad, but lists
```
019401|Audi A4 B9 (8W0)
019402|Audi A4 (8W0)
```
Anyone know the code for A4 B9 allroad, before I attempt these?

**Comment by Tomt97** [comment]
Anyone know the code setting for 18’ Q5

**Reply by Tomt97** [reply]
I have no PR code in the manual or spare wheel area

**Comment by Tomt97** [comment]
Nm found my old long code.   Copy it over and it work. 

But for Q5 it’s car class 4, Gen 2 , derivate 6, derivate supplement 0

**Comment by bwilliams75** [comment]
Unfortunately mine was a bit different issue. Had a little help trying to fix it and found out my MMX was bricked. New unit in the cards.

**Reply by Woody8219** [reply]
What was your issue? How did you fixed it? Best!

**Comment by PepeBarcelona** [comment]
Hi @Woody8219 , have a look to this post https://www.drive2.ru/l/684029549335300827/
There, you will see the solution that worked for me, but explained step by step.
The post is in Russian, but the google translation is quite accurate.
